### ISSUE TITLE
Escape key to close lightbox

### ISSUE THREAD

**Issue by jchwenger** [godaddy-wordpress/coblocks#1559]
The left/right arrow keys are linked to previous/next image, and it'd be lovely to press 'esc' to close the full-screen image version!

Cheers,
Jeremie

Coblocks: Version 2.0.2
WP: Version 5.4.2

**Comment by jrtashjian** [comment]
Thanks for the feature request @jchwenger! This should have been more obvious when we initially built in the lightbox functionality. Regardless this is a pretty straightforward fix we're going to get into the next release.

**Comment by jchwenger** [comment]
Grand!
